src/empty.js

```diff
@@ -3,7 +3,7 @@ var _hasMethod = require('./internal/_hasMethod');
 
 
 /**
- * `empty` wraps any object in an array. This implementation is compatible with the
+ * `empty` returns an empty list for any argument. This implementation is compatible with the
```

Comment:
I would say _`empty` returns the empty value of its argument's type_, since the empty value needn't be a list.